server/api/gmail_auth_handler.py

```diff
@@ -0,0 +1,80 @@
+from flask import jsonify, request, Blueprint, Response, session, url_for, redirect, current_app
+from flask_jwt_extended import get_jwt_identity, jwt_required
+from db.model import User
+from api.util import *
+from api import error_code
+from google_auth_oauthlib.flow import Flow
+from google.oauth2.credentials import Credentials
+from google.auth.transport.requests import AuthorizedSession
+import google.auth.exceptions
+
+gmail_auth_handler = Blueprint('gmail_auth_handler', __name__)
+
+SCOPES = [
+    'openid',
+    'https://www.googleapis.com/auth/gmail.readonly',
+    'https://www.googleapis.com/auth/userinfo.profile',
+    'https://www.googleapis.com/auth/gmail.compose',
+    'https://www.googleapis.com/auth/userinfo.email',
+    # Add other requested scopes.
+]
+
+CLIENTSECRETS_LOCATION = './credentials.json' # TODO: goes to config file
+# url_for('gmail_auth_handler.gmail_oauth_callback', _external=True)
+REDIRECT_URI = 'http://localhost:5000/gmail_oauth_callback'
+REDIRECT_URI_FRONT = 'http://localhost:3000/gmail_oauth_callback'
```

Comment:
can we make the beginning of these URIs (the domains) environment variables so we can change this later?